### PR TITLE
Alignment with other materials/adjustments

### PR DESCRIPTION
**Pull request by vortexing**

### CODE REVIEW

**Comment by vortexing** [comment]
Well I un-drafted this but it's not really ready for review.

File: 01-intro.Rmd [hunk 1]
```diff
@@ -64,17 +64,19 @@ For certain grant mechanisms for projects that do not generate data, compliance
...
-- [**Can I elect to not share data?**](#choose-to-not-share) The short answer is yes. However, you must still justify this choice in a submitted DMS Plan.
+- [**Does my research generate scientific data?**](#scientific-data) 
+- [**When do I need to share my data?**](#sharing-timeline) 
+- [**When should I not share data?**](#when-to-not-share) 
 
 To determine if your research requires compliance with other policies that may influence how you share your data, take this [quiz](https://sharing.nih.gov/other-sharing-policies/which-policies-apply-to-my-research).
 
 
-## During Grant Applications 
+## During grant applications
```
Comment (avahoffman) [inline 01-intro.Rmd:74]: let's keep the caps for consistency with other H2s throughout

File: 01-intro.Rmd [hunk 2]
```diff
@@ -64,17 +64,19 @@ For certain grant mechanisms for projects that do not generate data, compliance
...
 
 To determine if your research requires compliance with other policies that may influence how you share your data, take this [quiz](https://sharing.nih.gov/other-sharing-policies/which-policies-apply-to-my-research).
 
 
-## During Grant Applications 
+## During grant applications 
 
-### Which Grant Mechanisms {#no-sharing}
-🤔 What grant mechanisms don't require compliance with the DMS policy?
+### Specific grant mechanisms {#no-sharing}
```
Comment (avahoffman) [inline 01-intro.Rmd:76]:
```suggestion
### Specific Grant Mechanisms {#no-sharing}
```

File: 01-intro.Rmd [hunk 3]
```diff
@@ -109,8 +111,10 @@ ottrpal::include_slide("https://docs.google.com/presentation/d/1luFoDzF6aDJEebbL
 
 
 
-### Data Generating Research {#scientific-data}
-🤔 Does my research generate scientific data?
+### Data generating research {#scientific-data}
```
Comment (avahoffman) [inline 01-intro.Rmd:114]:
```suggestion
### Data-generating Research {#scientific-data}
```

File: 01-intro.Rmd [hunk 4]
```diff
@@ -152,11 +156,14 @@ You are not expected to share:
 <!-- </div> -->
 
 
-### What about grant renewals? 
+### Grant renewals
```
Comment (avahoffman) [inline 01-intro.Rmd:159]:
```suggestion
### Grant Renewals
```

File: 01-intro.Rmd [hunk 5]
```diff
@@ -171,18 +178,18 @@ For proposals where data sharing is specified as part of the Funding Opportunity
  - Program staff will also review the DMS plan.
  
 
-## Sharing timeline? {#sharing-timeline} 
+## Sharing timeline {#sharing-timeline}
```
Comment (avahoffman) [inline 01-intro.Rmd:181]:
```suggestion
## Sharing Timeline {#sharing-timeline} 
```

File: 01-intro.Rmd [hunk 6]
```diff
@@ -171,18 +178,18 @@ For proposals where data sharing is specified as part of the Funding Opportunity
  - Program staff will also review the DMS plan.
  
 
-## Sharing timeline? {#sharing-timeline} 
+## Sharing timeline {#sharing-timeline} 
 
 Data should be made available **no later than publication or end of the award**. This means that data underlying findings that are not published in peer-reviewed journals should be made available by the end of the award.
 
-### No cost extensions?
+### No cost extensions
```
Comment (avahoffman) [inline 01-intro.Rmd:185]:
```suggestion
### No-cost Extensions
```

File: 04-writing_a_DMS_plan.Rmd [hunk 7]
```diff
@@ -42,7 +43,7 @@ Keep in mind that your DMS plan will not be reviewed by reviewers, only the budg
 
 ☑️  Describe the expected amounts of data for each kind of data.
 
-☑️  Describe the expected levels of aggregation (individual, aggregated, summarized) of data for each kind of data.
+☑️  Describe the expected levels of aggregation (individual, aggregated, summarized) of data make sense to share for each kind of data.
```
Comment (avahoffman) [inline 04-writing_a_DMS_plan.Rmd:46]:
```suggestion
☑️  Describe the expected levels of aggregation (individual, aggregated, summarized) for each kind of data.
```

**Comment by carriewright11** [comment]
I think we need to merge main into her to get the question boxes - because I think I started that after this PR started.

**Comment by vortexing** [comment]
I'm-a-gonna merge in now.

**Pull request merged**